### PR TITLE
chore(deps): update ruff to 0.0.261

### PR DESCRIPTION
**Pull request by onerandomusername**
## Summary

bumps ruff to 0.0.261. Needed before another wave of prs to add more ruff rules.

Changelogs: 
https://github.com/charliermarsh/ruff/releases/tag/v0.0.261
https://github.com/charliermarsh/ruff/releases/tag/v0.0.260
https://github.com/charliermarsh/ruff/releases/tag/v0.0.259
https://github.com/charliermarsh/ruff/releases/tag/v0.0.258

## Checklist

- [ ] If code changes were made, then they have been tested
    - [ ] I have updated the documentation to reflect the changes
    - [ ] I have formatted the code properly by running `pdm lint`
    - [ ] I have type-checked the code by running `pdm pyright`
- [ ] This PR fixes an issue
- [ ] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: disnake/ext/commands/context.py [hunk 1]
```diff
@@ -348,7 +348,7 @@ async def send_help(self, *args: Any) -> Any:
             return None
 
         try:
-            entity.qualified_name
+            entity.qualified_name  # noqa: B018
         except AttributeError:
             # if we're here then it's not a cog, group, or command.
             return None
```
Comment (shiftinv) [inline disnake/ext/commands/context.py:354]: this should just use `hasattr`

File: tests/test_embeds.py [hunk 2]
```diff
@@ -479,4 +479,4 @@ def test_emptyembed() -> None:
 
     # make sure unknown module attrs continue to raise
     with pytest.raises(AttributeError):
-        embeds.this_does_not_exist  # type: ignore
+        embeds.this_does_not_exist  # type: ignore  # noqa: B018
```
Comment (shiftinv) [inline tests/test_embeds.py:482]: Instead of adding `noqa`s, `_ = embeds.this_does_not_exist` would make this explicit

**Review by shiftinv — CHANGES_REQUESTED** [review]

File: disnake/ext/commands/context.py (hunk 1 above)
Comment (onerandomusername) [reply disnake/ext/commands/context.py]: [`a071fd6` (#999)](https://github.com/DisnakeDev/disnake/pull/999/commits/a071fd6f15a8b5b540a7e605b46f99cd23ac86c6)

File: tests/test_embeds.py (hunk 2 above)
Comment (onerandomusername) [reply tests/test_embeds.py]: [`a071fd6` (#999)](https://github.com/DisnakeDev/disnake/pull/999/commits/a071fd6f15a8b5b540a7e605b46f99cd23ac86c6)

**Review by shiftinv — APPROVED** [review]

**Pull request merged**